app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<% flash.each do |key, value| %>
+  <div class="<%= flash_class(key) %>">
+    <%= value %>
+  </div>
+<% end %>
+
+<%= @current_admin %>
+<%= form_for(:session, url: admin_login_path) do |f| %>
```

Comment:
if we can, let's switch this to use `simple_form`  -- an example is in admin/problems/new